src/ForceHighPoly.Core/ForceHighPoly.cs

```diff
@@ -18,13 +17,15 @@ public partial class ForceHighPoly : BaseUnityPlugin
         public const string Version = "1.2.2";
 
         public static ConfigEntry<bool> Enabled { get; private set; }
+        public static ConfigEntry<bool> PartialPoly { get; private set; }
 
         internal void Main()
         {
             var hasEnoughRam = KKAPI.Utilities.MemoryInfo.GetCurrentStatus().ullTotalPhys > 16L * 1000L * 1000L * 1000L; // At least 16GB
             Enabled = Config.Bind("Config", "High poly mode", hasEnoughRam, "Whether or not to load high poly assets. Improves quality of characters in roaming mode and fixes some modded items not appearing.\nMay require exiting to main menu to take effect. This option has high memory requirements, at least 8GB of RAM is recommended (with ~10 characters).");
+            PartialPoly = Config.Bind("Config", "Partial High poly mode", true, "Use High Poly Assets on Characters who wear clothing that lack low poly models and prevents the resulting errors.\nMay require exiting to main menu to take effect. This option has Lower memory requirements, at least 8GB of RAM is recommended (with ~10 characters).");
```

Comment:
The partial setting is confusing, if I turn the 1st setting off and the second on does it load partially or not at all? Instead of adding a new setting it might be better to change type of the 1st setting to an enum that has None, Full, and Partial values, with partial being the default.

Reply:
The full version will take priority, but indeed a Enum works